runtime/ptrace_ios.cc

```diff
@@ -0,0 +1,99 @@
+// It is __imperative__ that the functions in this file are __not__ included in
```

Comment:
License block. You can copy the same from a sibling file.

Reply:
Done.